lib/manager/components/version/FeedVersionViewer.js

```diff
@@ -294,7 +294,8 @@ export type ListProps = Props
 
 class VersionList extends Component<ListProps> {
   render () {
-    return <Panel header={<h3>List of feed versions</h3>}>
+    return <Panel>
+      <Panel.Heading><Panel.Title componentClass='h3'>List of Feed Versions</Panel.Title></Panel.Heading>
```

Comment:
Also, could you check if this component renders when you switch from the "default" view for feed a version to the "List" view? (This problem does not exist in the dev branch.)

Reply:
I'm seeing the same problem but displaying itself in a different way. Instead of crashing like it does here it simply hides most of the UI.

I think this is related to the `FeedVersionSummary` work, I'll try to take a look...

Reply:
The feedVersions aren't loaded <img width="453" alt="Screen Shot 2022-05-06 at 7 39 36 AM" src="https://user-images.githubusercontent.com/86619099/167155460-46abb8f3-eddd-447a-abce-1b6162261bcf.png">